shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -114,6 +114,8 @@ - (void)await {
 }
 
 - (void)onDisplayLink:(CADisplayLink*)link {
+  TRACE_EVENT0("flutter", "VSYNC");
```

Comment:
Same here, the timestamp in the CADisplayLink (see `frame_start_time` below for the value adjusted to Flutter timestamps) is what should be recorded as the start of the current vertical sync window.

Reply:
See the other thread. We don't have means to set an explicit timestamp for sync events.